src/main/scala/mesosphere/marathon/core/storage/repository/TemplateRepositoryLike.scala

```diff
@@ -0,0 +1,58 @@
+package mesosphere.marathon
+package core.storage.repository
```

Comment:
Hm, I have to admit that I'm not sure if `TemplateRepository` and `BucketRepository` should be in the same package. The former is a different semantic meaning. Templates are unique to Marathon. The later is universal and not unique to Marathon. I'd put it with our ZooKeeper client files.